Platform/CommonBoardPkg/CfgData/Template_BootOption.dsc

```diff
@@ -50,7 +50,7 @@
 
     # !BSF NAME:{Hardware Partition}
     # !BSF TYPE:{Combo}
-    # !BSF OPTION:{0:User Partition, 1:Boot Partition 1, 2:Boot Partition 2}
+    # !BSF OPTION:{0:User Partition, 1:Boot Partition 1, 2:Boot Partition 2, 3:Boot Partition 3, 4:Boot Partition 4}

```

Comment:
Add more in HELP to indicate the meaning for 0-4  when Flash device is selected as boot device.